packages/terra-graphs/src/components/Combination/CombinationGraph.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import '../Graph.module.scss';
+import '../Line/LineGraph.module.scss';
+import '../Bar/BarGraph.module.scss';
+import '../PairedResult/PairedResultGraph.module.scss';
+
+const propTypes = {
+  /**
+   * ID for graph created.
+   */
+  graphID: PropTypes.string.isRequired,
+  /**
+   * Graph configuration object
+   */
+  graphConfig: PropTypes.object.isRequired,
+  /**
+   * data to be displayed in graph
+   */
+  dataset: PropTypes.arrayOf(PropTypes.object),
+  /**
+   * timeout to display multiple data contents in specific time interval.
+   */
+  timeout: PropTypes.arrayOf(PropTypes.number),
+};
+
+const datasetLoader = (dataset) => {
+  let graphType = dataset.graphType.toUpperCase();
+  graphType = graphType.replaceAll(' ', '');
+  let apiData;
+  switch (graphType) {
+    case 'LINE':
+      apiData = Carbon.api.line(dataset);
+      break;
+    case 'SCATTER':
+      apiData = Carbon.api.scatter(dataset);
+      break;
+    case 'PAIREDRESULT':
+      apiData = Carbon.api.pairedResult(dataset);
+      break;
+    case 'BAR':
+      apiData = Carbon.api.bar(dataset);
+      break;
+    case 'BUBBLESINGLEDATASET':
+      apiData = Carbon.api.bubbleSingleDataset(dataset);
+      break;
+    case 'BUBBLEMULTIPLEDATASET':
+      apiData = Carbon.api.bubbleMultipleDataset(dataset);
+      break;
+    default:
+      apiData = 'undefined';
```

Comment:
It looked like all the above Carbon.api return an object. Should apiData be `undefined` or a string type with a value of "undefined" if graphType is none of the above?

Reply:
It does not make much difference Ben. Whatever we pass other than the object(with necessary values) will lead to an error. I can change it if standards are missing here.

Reply:
If it does not have to be a string value of `undefined`, I think we should return `undefined` to avoid this function from returning different data types to minimize confusion.

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/150/commits/58f7cc18f44f83b8d0640e10439a30775ddf4698#diff-69b4098fd145f5ca79edeeb7a5fbeeec17981c0e5804acc94bca187ca411f4e0R52).